### DISCUSSION TITLE
How to get the status of modal ref

### DISCUSSION THREAD

**Discussion by MicahDavid** [gorhom/react-native-bottom-sheet#1666] [Q&A]
Is there a way to get the status of modal ref? I want to be able to know if the modal is presented or closed/dismissed.

**Comment by jonnyreeves** [comment]
+1, at present I am having to manage this state outside of bottom sheet, creating custom callback for `showBottomSheet` and `dismissBottomSheet`.

Ideally there would be an impereative function to get the presentation state of the BottomSheet, ie:

```
sheetRef.current.getSnapPointIndex();
sheetRef.current.isOpen();
```
